cmd/manager/main.go

```diff
@@ -174,6 +174,18 @@ func main() {
 
 	log.Print("Starting the Cmd.")
 
+	// migrate away from Marketplace API
+	clientGo, err := client.New(cfg, client.Options{Scheme: mgr.GetScheme()})
+	if err != nil {
+		log.Error(err, "Failed to instantiate client for migrator")
+		os.Exit(1)
+	}
+	migrator := migrator.New(clientGo)
+	err = migrator.Migrate()
+	if err != nil {
```

Comment:
this can be a followup pr, but if the migration fails we should probably retry it a couple of times.

(failing to migrate is not the worst, the crd will just stick around)

Reply:
@ecordell till now I haven't seen any error that could be non-deterministic. If we fail once, we're most likely going to fail all the time. I'm not sure if blocking the operator because we're trying multiple times is better than failing fast and reporting the error.